exporter/loadbalancingexporter/helpers_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package loadbalancingexporter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+)
+
+func TestMergeTracesTwoEmpty(t *testing.T) {
+	expectedEmpty := ptrace.NewTraces()
+	trace1 := ptrace.NewTraces()
+	trace2 := ptrace.NewTraces()
+
+	mergedTraces := mergeTraces(trace1, trace2)
+
+	require.Equal(t, expectedEmpty, mergedTraces)
+}
+
+func TestMergeTracesSingleEmpty(t *testing.T) {
+	expectedTraces := simpleTraces()
+
+	trace1 := ptrace.NewTraces()
+	trace2 := simpleTraces()
+
+	mergedTraces := mergeTraces(trace1, trace2)
+
+	require.Equal(t, expectedTraces, mergedTraces)
+}
+
+func TestMergeTraces(t *testing.T) {
```

Comment:
Can you also add a benchmark? I totally believe you that this brings better performance, but we should track this.

Reply:
If you decide to not do the benchmark, or if you prefer doing it on a follow-up PR, let me know.